EIPS/eip-1776.md

```diff
@@ -0,0 +1,94 @@
+
+ERC-1776 (Token Contract)
+interface ERC1776 extends ERC20 {
+  
+  /// @dev This emits when funds are minted
+  event AdminMint(address from, address to, uint256 amount);
+
+  /// @dev This emits when funds are burned
+  event AdminBurn(address from, uint256 amount);
+
+  /**
+  * @dev getter to determine how much of the collective fund is available to the user
+  */
+  function userFundBalance(address _operator) external view returns (number);
+
+}
+The ERC-1776 extends ERC-20. Due to the indivisible nature of asset tokens, the decimals number MUST be zero.
+
+Admin
+A key role is played by the Admin. This figure has the permission to manage Token Details. Token holders can remove an admin with 50+% and likewise an Admin can give their power to another. 
+
+Below is an example interface for the implementation of the Admin functionalities and follows closely Open Zepplin's Ownable contract with extended powers.
+
+Interface Admin {
+
+    function setName(string memory newName) public onlyOwner {
+        _name = newName;
+    }
+    function setSymbol(string memory newSymbol) public onlyOwner {
+        _symbol = newSymbol;
+    }
+    function setDocumentURI(string memory newURI) public onlyOwner {
+        documentURI = newURI;
+    }
+}
+
+Rationale
+Allowing a third party fixes many problems faced when tokenizing a real world asset (RWA) that needs to interface with existing legal jurisdictions.  Included is the metadata needed to interface with NFT standards with the definition of a DocumentURI.
+
+Backwards Compatibility
+The contract will have the ability to be somewhat backwards compatible with both ERC20 and ERC721 contracts just in different ways.  The transfer function will be ERC20 compatible, but will have a global DocumentURI field found with the standard ERC721 contract even though each succusive token can also have it's own such as in ERC1155
+This EIP does not introduce backward incompatibilities and is backward compatible with the older ERC-20 token standard. This standard allows the implementation of ERC-20 functions transfer, transferFrom, approve and allowance alongside to make a token fully compatible with ERC-20. The token MAY implement decimals() for backward compatibility with ERC-20. If implemented, it MUST always return 0.
+
+Implementations
+Although an official implementation of the ERC-1776 has not been presented, in the Specifications section some interfaces have been reported which, once implemented correctly and according to the reccomandations, allow to create an asset token compliant with this ERC.
+
+Security Considerations
+The security considerations mainly concern the role played by the Issuers. If they do not correctly manage their members and allow for profit through non-participation it could fall into a security classification although it is designed to be a utility DAO token.  As it is used and abuses found there may be fixes to prevent this.
+
+Copyright
+Business License 1.1
```

Comment:
All EIPs must be released into the public domain, see [EIP-1](https://eips.ethereum.org/EIPS/eip-1#copyright) for the required waiver.

Reply:
👍🏼